readme.md

```diff
@@ -20,9 +20,12 @@ There are three dependencies:
 ## Usage
 
 To make use of this project you can either add it as a submodule to your
-existing project, or copy the files into your repository. Now add the following
-to your `CMakeLists.txt`.
+existing project, or copy the files into your repository.
+It is __mandatory__ to set `GCF_TOP_LEVEL_DIR` to the directory of the git repository
```

Comment:
does not need to mandatory

Reply:
I'm still a bit torn on this. I want to make sure that it's not possible to accidentally miss this and thus create issues for downstream users. But I agree that it sacrifices the usability a bit.

Reply:
I see.
maybe you could add an example like passing `GCF_TOP_LEVEL_DIR = ${CMAKE_SOURCE_FOLDER}` if they need the original behavior into README